js/app.js

```diff
@@ -79,6 +81,20 @@ function headerRender() {
   trEl.appendChild(thEl);
   storeTable.appendChild(trEl);
 }
+function footerRender() {
+  var trEl = document.createElement('tr');
+  var thEl = document.createElement('th');
+  thEl.textContent = 'Column Totals';
+  trEl.appendChild(thEl);
+  for (var i = 0; i < hoursLong.length; i++) {
+    thEl = document.createElement('th');
+    thEl.textContent = columnTotals[i];
+    trEl.appendChild(thEl);
+  } thEl = document.createElement('th');
+  thEl.textContent = 'Total Daily Cookies';
+  trEl.appendChild(thEl);
+  storeTable.appendChild(trEl);
+}
 
 //Wednesday code: Event listeners and forms
 function updateBranch(event) {
```

Comment:
Since the user can enter a decimal value for the average sales, you should use `parseFloat` instead of `parseInt`. i.e.
```javascript
parseFloat(branchForm.avgSales.value)
```